app/env.js

```diff
@@ -1,10 +1,18 @@
-const path = require('path')
-const envalid = require('envalid')
-const dotenv = require('dotenv')
+import path from 'path'
+import envalid from 'envalid'
+import dotenv from 'dotenv'
+import { fileURLToPath } from 'url'
+import os from 'os'
+import fsWithSync from 'fs'
+
+const __filename = fileURLToPath(import.meta.url)
+const __dirname = path.dirname(__filename)
 
 /* istanbul ignore else */
 if (process.env.NODE_ENV === 'test') {
   dotenv.config({ path: 'test/test.env' })
+  const ipfsDir = fsWithSync.mkdtempSync(path.join(os.tmpdir(), 'dscp-ipfs-'))
+  process.env.IPFS_PATH = ipfsDir
```

Comment:
moved from `test/integration/fixtures.js` because it was failing to set `process.env.IPFS_PATH` after changing to use `import`

Reply:
import is asynchronous whilst require is synchronous